From 83204976c75e6855b1483d6d19f2de71af103265 Mon Sep 17 00:00:00 2001
From: Hariprasad Kelam <hkelam@marvell.com>
Date: Tue, 14 Jul 2020 21:27:24 +0530
Subject: [PATCH 01/11] octeontx2-af: Increase number of CGX interfaces

commit 46922d68230d3cc65bfaf1ffda8ea09c38932bc9 from
git@git.assembla.com:cavium/WindRiver.linux.git

98xx board has 5 CGX interfaces adjust CGX_MAX value accordingly.

Change-Id: Ide7a3952c4bb5ed35b148b0db82bfd58598f21f2
Signed-off-by: Hariprasad Kelam <hkelam@marvell.com>
Reviewed-on: https://sj1git1.cavium.com/c/IP/SW/kernel/linux/+/31918
Tested-by: sa_ip-sw-jenkins <sa_ip-sw-jenkins@marvell.com>
Reviewed-by: Sunil Kovvuri Goutham <Sunil.Goutham@cavium.com>
Signed-off-by: Kevin Hao <kexin.hao@windriver.com>
---
 drivers/net/ethernet/marvell/octeontx2/af/rvu.h | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/drivers/net/ethernet/marvell/octeontx2/af/rvu.h b/drivers/net/ethernet/marvell/octeontx2/af/rvu.h
index 6b95bdf3a0b4..ca6c546e1188 100644
--- a/drivers/net/ethernet/marvell/octeontx2/af/rvu.h
+++ b/drivers/net/ethernet/marvell/octeontx2/af/rvu.h
@@ -444,7 +444,7 @@ struct rvu_fwdata {
 #define FWDATA_RESERVED_MEM 1023
 	u64 reserved[FWDATA_RESERVED_MEM];
 	/* Do not add new fields below this line */
-#define CGX_MAX         4
+#define CGX_MAX         5
 #define CGX_LMACS_MAX   4
 	struct cgx_lmac_fwdata_s cgx_fw_data[CGX_MAX][CGX_LMACS_MAX];
 };
-- 
2.17.1

